differential_equations2.py

```diff
@@ -0,0 +1,40 @@
+#forward Euler method to numerically solve a logistic growth model
+#and then plots the solution using matplotlib
+from typing import Callable, List, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def f(t: float, u: float) -> float:
+    alpha = 0.2
+    R = 1.0
+    unext = alpha*(u) * (1-(u/R))
+    return unext
+u0 = 0.1
```

Comment:
I would make a function like this to run and plot you code:
```python
def run_and_plot(u0: float = 0.1, T: float = 40, N: int = 10) -> None:
    """Fill in with a sensible docstring."""
    t, u = forward_euler(f, u0, T, N)

    plt.plot(t, u)
    plt.xlabel('Time')
    plt.ylabel('Population')
    plt.title('Model for logistic growth')
    plt.show()
```

Reply:
Then you can just call `run_and_plot()` at the bottom of your script and it will run the function with the default values